lib/chat-utils/parse-commands-from-chat.js

```diff
@@ -29,7 +29,7 @@ function formatUnban(user) {
 
 function parseMessage(message) {
   const parsed = JSON.parse(message.replace('MSG ', ''));
-  return { user: parsed.nick, roles: parsed.features, message: parsed.data };
```

Comment:
I don't think this is correct. The format is definitely data, I've gotten thousands of messages from DGG chat and Haven't seen it defined as message. Can you show where this is the case?

Reply:
I'll double check, but at the time there was no data property at all so `undefined` got passed around resulting in a crash on the second message from any user.

Reply:
You were right, the bug was in the mock-chat not in the parser.